materialize-elasticsearch/driver.go

```diff
@@ -71,12 +71,12 @@ func (resource) GetFieldDocString(fieldName string) string {
 type driver struct{}
 
 func (driver) Spec(ctx context.Context, req *pm.SpecRequest) (*pm.SpecResponse, error) {
-	endpointSchema, err := jsonschema.Reflect(&config{}).MarshalJSON()
+	endpointSchema, err := schemagen.GenerateSchema("Elasticsearch Connection", &config{}).MarshalJSON()
 	if err != nil {
 		return nil, fmt.Errorf("generating endpoint schema: %w", err)
 	}
 
-	resourceSchema, err := jsonschema.Reflect(&resource{}).MarshalJSON()
+	resourceSchema, err := schemagen.GenerateSchema("Elasticsearch resource", &resource{}).MarshalJSON()
```

Comment:
It looks like in general these titles are title-cased, maybe `s/resource/Resource` here?

Reply:
Thanks for pointing that out. I actually changed this to `Elasticsearch Index`, since `Resource` felt too generic. I think that's more in line with the user experience we want, since if you're materializing to Elasticsearch, you probably already have an idea of what the entities are called in there.